plugins/plugin-sass/README.md

```diff
@@ -0,0 +1,35 @@
+# @snowpack/plugin-sass
+
+This plugin adds [Sass](https://sass-lang.com/) support to any Snowpack project. With this plugin, you can import any `*.scss` or `*.sass` Sass file from JavaScript and have it compile to CSS.
+
+This plugin also adds support for `.module.scss` Sass Modules. [Learn more.](https://www.snowpack.dev/#import-css-modules)
+
+## A Note on Sass Implementations
+
+Sass is interesting in that multiple compilers are available: [sass](https://www.npmjs.com/package/sass) (written in Dart) & [node-sass](https://www.npmjs.com/package/node-sass) (written in JavaScript). Both packages run on Node.js and both are popular on npm. However, [node-sass is now considered deprecated](https://github.com/sass/node-sass/issues/2952).
+
+**This plugin was designed to work with the `sass` package.** `sass` is automatically installed with this plugin as a direct dependency, so no extra effort is required on your part.
```

Comment:
Sidenote: maybe we should set up Dependabot for Snowpack, at least to prompt version pushes? While being always up-to-date isn’t possible, we should keep this reasonably up-to-date, especially with all the recent changes/improvements to Sass (it’s had a lot of development this year again, after a period of stagnation)

Reply:
We're actually really lax about which version of Sass that we take here, so it would take a full major Sass update for us to get out of date. But yea, I'm not opposed to trying out a very quiet, major-only dependabot setup 👍